spec/system/admin/bulk_order_management_spec.rb

```diff
@@ -790,7 +790,7 @@
       let!(:li1) { create(:line_item_with_shipment, order: o1 ) }
       let!(:li2) { create(:line_item_with_shipment, order: o2 ) }
       let!(:p3) {
-        create(:product_with_option_types, group_buy: true, group_buy_unit_size: 5000,
+        create(:product_with_option_types, group_buy: true, group_buy_unit_size: 5,
```

Comment:
I have a question here actually.

Once the product unit has been set to `kg` : 
<img width="263" alt="Capture d’écran 2021-12-06 à 10 38 42" src="https://user-images.githubusercontent.com/296452/144823009-0484ba38-466c-4ba8-8a42-c001c63a8e06.png">

If we want to set the bulk unit size to `5kg` should the `Bulk Unit Size` field be 5 (`kg`) or 5000 (`g`)?
<img width="527" alt="Capture d’écran 2021-12-06 à 10 38 24" src="https://user-images.githubusercontent.com/296452/144822952-592baff5-e49f-4dee-96c7-90c4c5736c5a.png">

My understanding is that it should be `5`, but I'm wondering which values are set by our users to make the Bulk Order Management interface works:

 
<img width="1379" alt="Capture d’écran 2021-12-06 à 10 42 29" src="https://user-images.githubusercontent.com/296452/144823615-53f55d91-7047-4e40-a9df-5cfce65c79fd.png">

 Please help @openfoodfoundation/train-drivers-product-owners ... 🙏

Reply:
This is what the user guide says: 

![image](https://user-images.githubusercontent.com/6525576/144881735-3685d803-841a-4825-af6d-0bbe192aa402.png)


So the correct answer is 5000 actually :+1: